docs/book/src/networking-introduction.md

```diff
@@ -0,0 +1,98 @@
+# Networking Introduction
+
+In this chapter, you will find a very short introduction to various networking concepts. 
+These concepts are important to understanding when to use QUIC.
+
+## 1. TCP/IP and UDP Comparison
+
+Lets define some transport guarantees and their properties:
+
+| Transport Guarantees | Packet Loss [(1)][1]  | Packet Duplication [(2)][2] | Packet Order | Packet Delivery |
+| :-------------:   | :-------------: | :-------------: | :-------------: | :-------------:
+|   **Unreliable**  |       Any       |      Yes        |     No          |   No
+|   **Reliable**    |       No        |      No         |     Ordered     |   Yes
```

Comment:
Still not a fan of this table -- I feel it conflates too many concepts. I'd prefer a concise bullet list explanation of unreliable, unordered (UDP) and reliable, ordered (TCP).

Reply:
In this bullet point list, do you still want to mention those points?

Reply:
I think only loss and order matter -- these are related; the other things fall out of these.